### ISSUE TITLE
omg real aeroantuics!!!!1!

### ISSUE THREAD

**Issue by PouPeuu** [Modders-of-Create/Create-Aeronautics#2]
![dog](https://github.com/Modders-of-Create/Create-Aeronautics/assets/80326018/da4f3e29-573f-4838-9ce1-27a3a860a2c9)
![cat look inside](https://github.com/Modders-of-Create/Create-Aeronautics/assets/80326018/2b502af7-9890-48d4-9807-140cc36a3e8c)

**Comment by samuelh2005** [comment]
No, its just a template mod that does nothing.

**Comment by PouPeuu** [comment]
> No, its just a template mod that does nothing.

ik lol this is a joke
why is it called create aeronautics tho?

**Comment by samuelh2005** [comment]
> > No, its just a template mod that does nothing.
> 
> ik lol this is a joke why is it called create aeronautics tho?

Not sure, but I've reported this repository to the Create Aeronautics discord.

More information in the following thread: https://discord.com/channels/937435293294919690/1189150185700474880

**Comment by PouPeuu** [comment]
> > > No, its just a template mod that does nothing.
> > 
> > ik lol this is a joke why is it called create aeronautics tho?
> 
> Not sure, but I've reported this repository to the Create Aeronautics discord.
> 
> More information in the following thread: https://discord.com/channels/937435293294919690/1189150185700474880

Im fairly certain that they dont care

**Comment by samuelh2005** [comment]
> > > > No, its just a template mod that does nothing.
> > > 
> > > 
> > > ik lol this is a joke why is it called create aeronautics tho?
> > 
> > 
> > Not sure, but I've reported this repository to the Create Aeronautics discord.
> > More information in the following thread: https://discord.com/channels/937435293294919690/1189150185700474880
> 
> Im fairly certain that they dont care

looks like they deleted the thread now.

**Comment by PouPeuu** [comment]
> > > > > No, its just a template mod that does nothing.
> > > > 
> > > > 
> > > > ik lol this is a joke why is it called create aeronautics tho?
> > > 
> > > 
> > > Not sure, but I've reported this repository to the Create Aeronautics discord.
> > > More information in the following thread: https://discord.com/channels/937435293294919690/1189150185700474880
> > 
> > Im fairly certain that they dont care
> 
> looks like they deleted the thread now.

Yeah, they aren't very fond of people sending random links to "Create Aeronautics mod"

**Comment by samuelh2005** [comment]
It seem like there is another repository with real Create_Aeronautics code, https://github.com/gamerman2354235/Create_Aeronautics

**Comment by samuelh2005** [comment]
I've reported this repository and the other to Google.

**Comment by PouPeuu** [comment]
> It seem like there is another repository with real Create_Aeronautics code, https://github.com/gamerman2354235/Create_Aeronautics

that's legacy aero before they went private

**Comment by ff6f8d68** [comment]
> > It seem like there is another repository with real Create_Aeronautics code, https://github.com/gamerman2354235/Create_Aeronautics
> 
> that's legacy aero before they went private

legacy aero is better than nothing
